src/Administration/Resources/app/administration/src/module/sw-profile/snippet/de-DE.json

```diff
@@ -28,15 +26,7 @@
       "notificationSaveSuccessMessage": "Deine Profilinformationen wurden erfolgreich gespeichert.",
       "notificationSaveErrorMessage": "Deine Profilinformationen konnten nicht gespeichert werden.",
       "labelUploadAvatar": "Lade ein Profilbild hoch",
-      "labelUseAsAvatar": "Als Profilbild nutzen",
-      "placeholderNewPasswordConfirm": "Wiederhole Dein neues Passwort ...",
-      "placeholderNewPassword": "Gib ein neues Passwort ein ...",
-      "placeholderOldPassword": "Gib Dein altes Passwort ein ...",
-      "titleConfirmPasswordModal": "Passwort bestätigen",
-      "labelConfirmPassword": "Passwort",
-      "placeholderConfirmPassword": "Gib Dein aktuelles Passwort ein ...",
-      "labelButtonCancel": "Abbrechen",
-      "labelButtonConfirm": "Bestätigen"
```

Comment:
May you explain a little bit here why nine keys were deleted, but this number in the `en-GB.json` file is only 6 🤔

Reply:
This is only one thing left 😉